docs/api.html

```diff
@@ -165,3 +165,22 @@
 
 Examples showing how to use the consumer are given in the
 <a href="http://kafka.apache.org/0100/javadoc/index.html?org/apache/kafka/clients/consumer/KafkaConsumer.html" title="Kafka 0.9.0 Javadoc">javadocs</a>.
+
+<h3><a id="streamsapi" href="#streamsapi">2.3 Streams API</a></h3>
+
+As of the 0.10.0 release we have added a new client library named <b>Kafka Streams</b> to let users implement their stream processing
+applications with data streams stored in Kafka. This client is considered alpha quality and its public APIs are likely to change in
```

Comment:
"data streams stored in Kafka" -> "data stored in Kafka topics"  (until the introduction of Kafka Streams the terms "streams" or "data streams" have not been part of Kafka's terminology).